Formula/gambit-scheme.rb

```diff
@@ -24,32 +24,41 @@ class GambitScheme < Formula
     sha256 x86_64_linux:   "279db92ba64c71c31bf9a57df2414b0b47f497b47fd1c7a2fc39657be3b47db4"
   end
 
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
+
+  on_macos do
+    depends_on "gcc"
```

Comment:
No linkage on macOS. Probably just pure C code. In which case this can probably just be `:build`-only.

Reply:
It saves the compiler as default in config file (i.e. `gcc-13`). The GCC optimizations also add some GCC-specific flags.

Reply:
Ok, fair enough.

Reply:
Specifically, file `bin/gambcomp-C` sets up default values:
```
# The following settings are determined by the configure script.

C_COMPILER="gcc-13"
C_PREPROC="gcc-13 -E"

FLAGS_OBJ=" -Wno-deprecated-declarations -I\"/usr/local/opt/openssl@3/include\"   -Wno-unused -Wno-write-strings -Wdisabled-optimization -fwrapv -fno-strict-aliasing -fno-trapping-math -fno-math-errno -fschedule-insns2 -fmodulo-sched -freschedule-modulo-scheduled-loops -fomit-frame-pointer -fPIC -fno-common "
FLAGS_DYN="  -Wno-deprecated-declarations -I\"/usr/local/opt/openssl@3/include\"  -bundle  -Wno-unused -Wno-write-strings -Wdisabled-optimization -fwrapv -fno-strict-aliasing -fno-trapping-math -fno-math-errno -fschedule-insns2 -fmodulo-sched -freschedule-modulo-scheduled-loops -fomit-frame-pointer -fPIC -fno-common   -flat_namespace -undefined suppress"
FLAGS_LIB="  -Wno-deprecated-declarations -I\"/usr/local/opt/openssl@3/include\"  -dynamiclib -install_name \$(libdir)/\$(LIBRARY)\$(LIB_VERSION_SUFFIX)   -flat_namespace -undefined suppress"
FLAGS_EXE="  -Wno-deprecated-declarations -I\"/usr/local/opt/openssl@3/include\"   -Wno-unused -Wno-write-strings -Wdisabled-optimization -fwrapv -fno-strict-aliasing -fno-trapping-math -fno-math-errno -fschedule-insns2 -fmodulo-sched -freschedule-modulo-scheduled-loops -fomit-frame-pointer -fPIC -fno-common  "

FLAGS_OPT=" -O1"
FLAGS_OPT_RTS=" -O3"
```

In separate PR, I may add a note and maybe fix up OpenSSL path as above shows `/usr/local` when I compiled on ARM. Only may cause an issue if someone also has Rosetta.

Reply:
Let's see if we can fix that already now...